src/components/ingredients/CreateIngredientInput.tsx

```diff
@@ -41,66 +40,61 @@ const CreateIngredientInput: FC<Props> = ({
 
   return (
     <Flex alignItems={"center"} flexDir={"column"}>
-      <Box
-        bg={"rgb(242, 244, 244)"}
-        py={"32px"}
-        mb={"48px"}
-        rounded={"3xl"}
-        display={"flex"}
-        justifyContent={"center"}
-        width="-webkit-fill-available"
-        zIndex={0}
-      >
-        <Box ref={dropdownRef}>
-          <Input
-            width={"600px"}
-            onFocus={handleFocus}
-            height={"40px"}
-            rounded={"xl"}
-            fontSize={"14px"}
+      <Box ref={dropdownRef}>
+        <Input
+          width={"740px"}
+          onFocus={handleFocus}
+          height={"40px"}
+          rounded={"xl"}
+          fontSize={"14px"}
+          bg={"white"}
+          color={"rgb(132, 140, 145)"}
+          outline={"none"}
+          border={"1px solid rgb(132, 140, 145)"}
+          placeholder={
+            "Vyhledejte produkty, které odpovídají vaším požadavkům na danou ingredienci."
+          }
+          value={searchQuery}
+          onChange={(e) => setSearchQuery(e.target.value)}
+        />
+        {productsByIds &&
+        Object.keys(productsByIds).length > 0 &&
+        isDropdownOpen ? (
+          <Box
+            zIndex={1}
+            width={"740px"}
+            maxHeight={"480px"}
+            overflow="scroll"
+            sx={{
+              "::-webkit-scrollbar": {
+                color: "#007 #bada55",
+              },
+            }}
+            border={"1px solid rgb(218, 222, 224)"}
+            rounded={"lg"}
+            boxShadow={"md"}
+            position={"absolute"}
             bg={"white"}
-            color={"rgb(132, 140, 145)"}
-            outline={"none"}
-            border={"1px solid rgb(132, 140, 145)"}
-            placeholder={
-              "Vyhledejte produkty, které odpovídají vaším požadavkům na danou ingredienci."
-            }
-            value={searchQuery}
-            onChange={(e) => setSearchQuery(e.target.value)}
-          />
-          {productsByIds &&
-          Object.keys(productsByIds).length > 0 &&
-          isDropdownOpen ? (
-            //close on outside click
-            <Box
-              width={"600px"}
-              maxHeight={"480px"}
-              overflow="scroll"
-              sx={{
-                "::-webkit-scrollbar": {
-                  color: "#007 #bada55",
-                },
-              }}
-              border={"1px solid rgb(218, 222, 224)"}
-              rounded={"lg"}
-              boxShadow={"md"}
-              position={"absolute"}
-              bg={"white"}
-            >
-              {productIds.map((productId) => {
-                console.log(productsByIds, productId);
+          >
+            {productIds
+              .filter(
+                (productId) =>
+                  !selectedProducts.find((product) => product.id === productId),
+              )
+              .map((productId) => {
```

Comment:
Zase ciste stylistika, ale kdybych mel zachovat tuhle flow, vytahnul bych si to do promene. 

```
const XYZ = productIds
              .filter(
                (productId) =>
                  !selectedProducts.find((product) => product.id === productId),
              );
              
XYZ.(productId) => <>...</>
```

Na prvni dobrou je to jedno, ale co povazuju za super vyhodu je, ze te to nuti to pres co mapujes pojmenovat, coz pak hodne napomaha maintabilite toho kodu. Je na prvni dobrou zrejmy, co to ma delat.